Airports/EGNV/Sector.txt

```diff
@@ -3,7 +3,7 @@ DISPLAY:NVAPP:NVAPP:NVTWR
 DISPLAY:NVTWR:NVAPP:NVTWR
 
 SECTORLINE:NVAPP
-DISPLAY:East:East:NVAPP
+DISPLAY:NVAPP:East:NVAPP
 DISPLAY:NVAPP:East:East
```

Comment:
This will never active. East and East will always be the same person

Reply:
Not quite the resolution that was required, I'll comment what it needs to be